api/labels.py

```diff
@@ -0,0 +1,11 @@
+from maxfw.core import MetadataAPI
+
+
+class ModelLabelsAPI(MetadataAPI):
+    def get(self):
+        """Return the labels associated with the model"""
+
+        labels = ['background', 'car', 'motorbike', 'aeroplane', 'cat', 'person', 'bicycle', 'chair', 'pottedplant',
```

Comment:
would it be possible to put this in the order of the object ID (i.e. ['background', ''aeroplane', 'bicycle', ...]) so a label's index corresponds to its ID as listed in the README? that makes it easier and more valuable to developers. for example, if they have a segment with ID `3` then can just go to `labels[3]` and get the correct label (`bird`).

Reply:
Wouldn't it be better to attach the label [id] to each segment map element to mimic the Object Detector behavior?
```
[
    {
      "label_id": "1",
      "label": "person",
      "segment": [ ...]
      ]
    }
```

Reply:
@vabarbos I meant to put this in order, I copied and pasted from the table. I will fix this.

@ptitzler I'm slightly against that because unlike the object detector which will only return a small number of predictions this model will return thousands of predictions

Reply:
Never mind my comment. I misunderstood the returned data structure. What you have is fine.